libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs

```diff
@@ -75,13 +76,11 @@ newtype ScriptHash crypto
   deriving (Show, Eq, Ord, Generic)
   deriving newtype (NFData, NoThunks)
 
-deriving newtype instance
-  CC.Crypto crypto =>
-  ToCBOR (ScriptHash crypto)
+deriving newtype instance CC.Crypto crypto => Keyed (ScriptHash crypto)
 
-deriving newtype instance
-  CC.Crypto crypto =>
-  FromCBOR (ScriptHash crypto)
+deriving newtype instance CC.Crypto crypto => ToCBOR (ScriptHash crypto)
+
+deriving newtype instance CC.Crypto crypto => FromCBOR (ScriptHash crypto)
```

Comment:
Where's the underlying `Keyed` instance coming from?

Reply:
Ah, added in this PR :+1:

Reply:
Yep :+1: